packages/EasyCore/tests/Bridge/Symfony/Serializer/TrimStringsDenormalizerTest.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Tests\Bridge\Symfony\Serializer;
+
+use EonX\EasyCore\Bridge\Symfony\Serializer\TrimStringsDenormalizer;
+use EonX\EasyCore\Helpers\StringsTrimmerInterface;
+use EonX\EasyCore\Tests\Bridge\Symfony\AbstractSymfonyTestCase;
+use Mockery\MockInterface;
+use stdClass;
+use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
+
+/**
+ * @covers \EonX\EasyCore\Bridge\Symfony\Serializer\TrimStringsDenormalizer
+ *
+ * @internal
+ */
+final class TrimStringsDenormalizerTest extends AbstractSymfonyTestCase
+{
+    /**
+     * @return mixed[]
+     *
+     * @see testSupportsDenormalizationReturnsTrue
+     */
+    public function provideCorrectDataForDenormalization(): array
```

Comment:
Let's change to `provideDataForSupportsDenormalization` and combine with `provideIneligibleDataForDenormalization`. And use all of the sets in one `testSupportsDenormalizationReturnsExpectedResult`.

Reply:
Done.